<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtTree</class>
 <widget class="QMainWindow" name="QtTree">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtTree</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>638</width>
         <height>479</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QLabel" name="labPic">
          <property name="text">
           <string>pic</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>959</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>目录树</string>
    </property>
    <addaction name="actionAddDir"/>
    <addaction name="actionAddFile"/>
    <addaction name="actionDelNode"/>
    <addaction name="actionLoopNode"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionActual"/>
    <addaction name="separator"/>
    <addaction name="actionFitWidth"/>
    <addaction name="actionFitHeight"/>
    <addaction name="actionFloat"/>
    <addaction name="actionShow"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>110</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>524287</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <column>
        <property name="text">
         <string>节点</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>类型</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>其他</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>root</string>
        </property>
        <property name="text">
         <string>root</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>A</string>
        </property>
        <property name="text">
         <string>dir</string>
        </property>
        <item>
         <property name="text">
          <string>a</string>
         </property>
         <property name="text">
          <string>file</string>
         </property>
        </item>
       </item>
       <item>
        <property name="text">
         <string>B</string>
        </property>
        <property name="text">
         <string>file</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddDir"/>
   <addaction name="actionAddFile"/>
   <addaction name="actionDelNode"/>
   <addaction name="actionLoopNode"/>
   <addaction name="separator"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionActual"/>
   <addaction name="separator"/>
   <addaction name="actionFitWidth"/>
   <addaction name="actionFitHeight"/>
   <addaction name="actionFloat"/>
   <addaction name="actionShow"/>
  </widget>
  <action name="actionAddDir">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_tree/tianjiamulu_1.svg</normaloff>:/icon_tree/tianjiamulu_1.svg</iconset>
   </property>
   <property name="text">
    <string>添加目录</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionAddFile">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_tree/tianjiawenjian_1.svg</normaloff>:/icon_tree/tianjiawenjian_1.svg</iconset>
   </property>
   <property name="text">
    <string>添加文件</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionDelNode">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_tree/jiedian.svg</normaloff>:/icon_tree/jiedian.svg</iconset>
   </property>
   <property name="text">
    <string>删除节点</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionLoopNode">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_tree/shituxianshiquanbujiedian.svg</normaloff>:/icon_tree/shituxianshiquanbujiedian.svg</iconset>
   </property>
   <property name="text">
    <string>遍历节点</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_tree/fangda.svg</normaloff>:/icon_tree/fangda.svg</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_tree/suoxiao.svg</normaloff>:/icon_tree/suoxiao.svg</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionActual">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_tree/fangdajing_1.svg</normaloff>:/icon_tree/fangdajing_1.svg</iconset>
   </property>
   <property name="text">
    <string>实际大小</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionFitWidth">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_tree/fuwei_1.svg</normaloff>:/icon_tree/fuwei_1.svg</iconset>
   </property>
   <property name="text">
    <string>适应宽度</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionFitHeight">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_tree/fuwei.svg</normaloff>:/icon_tree/fuwei.svg</iconset>
   </property>
   <property name="text">
    <string>适应高度</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionFloat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_tree/chuangkoufangda.svg</normaloff>:/icon_tree/chuangkoufangda.svg</iconset>
   </property>
   <property name="text">
    <string>窗口浮动</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionShow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_tree/chuangkou.svg</normaloff>:/icon_tree/chuangkou.svg</iconset>
   </property>
   <property name="text">
    <string>窗口可见</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
